src/test/java/com/powsybl/openloadflow/ac/GeneratorRemoteControlTest.java

```diff
@@ -580,6 +580,22 @@ void testNotSupportedRemoteReactivePowerControl() {
         assertReactivePowerEquals(0.274417, l34.getTerminal(TwoSides.TWO));
     }
 
+    @Test
+    void testGeneratorUpdateWithReactivePowerControlDisabled() {
+        // control is off through parameters
+        Network network2 = FourBusNetworkFactory.createWithReactiveControl2GeneratorsOnSameBus();
+        Generator g1n1 = network2.getGenerator("g1Bis");
+        g1n1.setTargetQ(5.0);
+        Generator g1n2 = network2.getGenerator("g1");
+        g1n2.setTargetQ(3.0);
+        Line l34n2 = network2.getLine("l34");
+        LoadFlowResult result2 = loadFlowRunner.run(network2, parameters);
+        assertTrue(result2.isOk());
+        assertReactivePowerEquals(-0.287, l34n2.getTerminal(TwoSides.TWO));
+        assertReactivePowerEquals(-5.0, g1n1.getTerminal());
+        assertReactivePowerEquals(-3.0, g1n2.getTerminal());
+    }
```

Comment:
Maybe add a condition to check the mismatch on the node with the two generators ?

```java
assertEquals(0.0, g1n1.getTerminal().getQ() +
      g1n2.getTerminal().getQ() +
      network2.getLine("l12").getTerminal1().getQ() +
      network2.getLine("l14").getTerminal1().getQ() +
      network2.getLine("l13").getTerminal1().getQ(),
      0.01);
```

Reply:
It is verified on main branch, so I removed it.